app/models/agent.rb

```diff
@@ -26,6 +26,9 @@ class Agent < ApplicationRecord
   scope :active, -> { where(deleted_at: nil) }
   scope :order_by_last_name, -> { order(Arel.sql('LOWER(last_name)')) }
   scope :secretariat, -> { joins(:service).where(services: { name: 'Secrétariat'.freeze }) }
+  scope :available_for_rdv, lambda { |motif, service|
```

Comment:
by the way, no need to pass both the motif and the service as parameters as it's the service's motif that's used